src/tools/generators.ts

```diff
@@ -354,6 +354,31 @@ export async function generateFromTemplate(
   return { written, exists, overwritten }
 }
 
+/**
+ * Checks a file for front matter.
```

Comment:
We can make this more accurate, this function actually is looking for `destinationDir` to be present correct? Not just any front matter in the file